R/AutoTuner.R

```diff
@@ -41,16 +41,27 @@
 #' terminator = trm("evals", n_evals = 5)
 #' tuner = tnr("grid_search")
 #' at = AutoTuner$new(
-#'   learner, resampling, measure, search_space, terminator,
-#'   tuner, store_tuning_instance = TRUE)
+#'   learner = learner,
+#'   resampling =  resampling,
+#'   measure =  measure,
+#'   terminator = terminator,
+#'   tuner = tuner,
+#'   search_space = search_space,
+#'   store_tuning_instance = TRUE)
```

Comment:
imho this is a bit verbose for example code where every argument has exactly the right name already. Are you adding this for the sake of consistency with something? I guess its good practice to call functions with many arguments like this, but I think it subtracts from the example code qua example code.